### ISSUE TITLE
Observable.catch() a bit misleading

### ISSUE THREAD

**Issue by mateuszkocz** [btroncone/learn-rxjs#21]
Thanks for creating this great resource! While learning I found a small issue with the `catch` operator's description.

```
TL;DR: Gracefully handle errors and exceptions **without terminating** observable

The catch operator allows us to capture errors and exceptions, returning an observable.
Properly caught errors **will avoid terminating** a subscription chain.
```

It suggests the subscription will stay working after the error is thrown in the Observable, but it's not what will really happen. The error will terminate the original Observable anyway. The effect of using the `catch` operator is that it will trigger the `onNext` callback, rather than `onError` before executing the `onFinish` and ending the subscription.

http://jsbin.com/diwaheyawu/edit?js,console

**Comment by btroncone** [comment]
@mateuszkocz Yep, you are correct, thanks for the call out! I will update this shortly. 👍

**Comment by btroncone** [comment]
This should now be fixed, thanks again!
